definitions/npm/@apollo/react-hooks_v3.x.x/CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+# React Apollo
+
+## Common reviewers
+
+Below is a list of common reviewers on Apollo related libraries:
+
+- wzrdzl
+- thymikee
+- rasmusprentow
+- mic
+- bwlt
+- TLadd
+- budde377
+- jedwards1211
```

Comment:
Copy-pasta for this file?

Reply:
I copied over reviewers for the other apollo related packages. It was intentional, but I can change it if you want